CHANGELOG.md

```diff
@@ -6,9 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0] - 2022-10-03
+### Changed
```

Comment:
This PR is missing: https://github.com/MetaMask/json-rpc-middleware-stream/pull/15

Specifically, the change in minimum supported Node.js version is notable, because it's a breaking change.

Reply:
Looks good, except it should be prefixed with "BREAKING:"

Reply:
Yeah, good point.

Reply:
@jpuri could you add the "BREAKING:" prefix, so that people know which change is breaking?

Reply:
PR is updated